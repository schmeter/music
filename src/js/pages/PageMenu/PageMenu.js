import React, { useEffect } from 'react';

import Page from '../../components/Page';
import Menu from '../../components/Menu';
import Ball from '../../components/Ball';
import { setTitle } from '../../services/meta';

const PageMenu = () => {
  useEffect(() => {
    setTitle();
  }, []);

  return (
    <Page id="menu">
      <Menu />
      <Ball />
    </Page>
  );
};

PageMenu.propTypes = {};

export default PageMenu;
