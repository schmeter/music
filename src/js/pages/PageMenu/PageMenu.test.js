import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import PageMenu from './PageMenu';

// eslint-disable-next-line react/display-name
jest.mock('../Page', () => () => <div />);
// eslint-disable-next-line react/display-name
jest.mock('../../components/Menu', () => () => <div />);

describe('PageMenu', () => {
  it('renders correctly', () => {
    const props = {
      activeTrack: {
        album: {
          imgPath: 'test',
        },
      },
    };
    const component = mount(<PageMenu {...props} />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
