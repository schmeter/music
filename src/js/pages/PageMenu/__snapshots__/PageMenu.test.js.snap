// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageMenu renders correctly 1`] = `
<PageMenu
  activeTrack={
    Object {
      "album": Object {
        "imgPath": "test",
      },
    }
  }
>
  <Component
    id="menu"
  >
    <div />
  </Component>
</PageMenu>
`;
